include/gz/utilities/ExtraTestMacros.hh

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef GZ_UTILITIES_EXTRATESTMACROS_HH
+#define GZ_UTILITIES_EXTRATESTMACROS_HH
+
+#include <gz/utilities/detail/ExtraTestMacros.hh>
+
+#pragma message("ign-cmake (utilities) ExtraTestMacros is deprecated, use ign-utils")
```

Comment:
Since this is deprecated anyway, how about we don't add `gz` aliases for them? That is, users of `ignition/utilities` shouldn't migrate to `gz/utilities`, but to `gz/utils`, which is in another library.